tests/SampleData.hs

```diff
@@ -193,6 +207,20 @@ flavorTimedPDsResult :: [(PFValue PDInstanceFlavor, Word64)]
 flavorTimedPDsResult = [ (flavorID1, 9)
                        , (flavorID2, 4) ]
 
+propSafetyInstance :: [Timed PDInstanceFlavor] -> Bool
+propSafetyInstance x =
+    let testPred (PDInstanceFlavor InstanceActive _) = True
+        testPred _                                   = False
+        fullResult     = L.fold (foldInstanceFlavor (const True)) x
+        filteredResult = L.fold (foldInstanceFlavor testPred    ) x
+        fullKeys       = M.keys fullResult
+        filteredKeys   = M.keys filteredResult
+    in  subset filteredKeys fullKeys &&
+        all (\k -> isJust $ do
+                v  <- M.lookup k fullResult
+                v' <- M.lookup k filteredResult
+                return $ v' <= v) filteredKeys
+
```

Comment:
can you put a short comment describing this property -- something to the effect of "filtered results must be subset of ... and all filtered values are less than corresponding values in full results"

then please merge.